### PR TITLE
Project atomic blog: docker lvm plugin

### PR DESCRIPTION
**Pull request by shishir-a412ed**
Signed-off-by: Shishir Mahajan <shishir.mahajan@redhat.com>

### CODE REVIEW

**Comment by shishir-a412ed** [comment]
ping @jberkus @bproffitt

**Comment by jberkus** [comment]
Feh, looks like CI is down again.  Grrr.

**Comment by jberkus** [comment]
@bproffitt this looks good to me.  Please publish either Monday or Tuesday depending on when the other blog post goes out.

**Comment by bproffitt** [comment]
Posted this one first.

**Comment by shishir-a412ed** [comment]
thanks ! @jberkus @bproffitt for the quick response.

**Comment by jberkus** [comment]
Thank you for the blog post!

File: source/blog/2016-04-29-docker-lvm-plugin.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+ 
+# <span style="color:cornflowerblue">docker lvm plugin</span>
+## <span style="color:cornflowerblue">Author: Shishir Mahajan</span>
+---
+Recently docker introduced a volume command (docker volume –help for more information) to allow users to create a logical volume and then bind mount it inside the container at container creation/run time.  You can achieve this by:
```
Comment (runcom) [inline source/blog/2016-04-29-docker-lvm-plugin.md]: *Recently* is kind of wrong since `docker volume` is in docker since 1.9.x (we're now about at 1.12)

**Pull request merged**